src/components/calendar/calendar.e2e.ts

```diff
@@ -11,6 +11,11 @@ import { SbbCalendarElement } from './calendar';
 describe('sbb-calendar', () => {
   const selected = new Date(2023, 0, 15).getTime() / 1000;
   let element: SbbCalendarElement;
+  const waitForTransition = async (): Promise<void> => {
+    await waitForCondition(() => element.hasAttribute('data-transition'));
+    await waitForLitRender(element);
+    await waitForCondition(() => !element.hasAttribute('data-transition'));
+  };
```

Comment:
I have some doubts that this runs stable, as adding and removing of data-transition could be done faster than the waitforcondition waits.

Reply:
could it be that disableAnimation property is missing on the calender?